dev-lang/cobol/gnucobol-3.0~rc1.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A free COBOL compiler"
+DESCRIPTION="GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler."
+HOMEPAGE="https://sourceforge.net/projects/open-cobol/"
+SOURCE_URI="https://downloads.sf.net/open-cobol/gnu-cobol/gnucobol-3.0-rc1.tar.gz"
+REVISION="1"
+LICENSE="GNU GPL v3"
+COPYRIGHT="2001-2019 Free Software Foundation, Inc"
+CHECKSUM_SHA256="e55aeea6b1f77e763b4cd4b520c78eb06da7671b4dcc76463fd0237dbf2e4816"
+SOURCE_DIR="gnucobol-3.0-rc1"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	gnucobol = $portVersion
+	cmd:cob_config
+	cmd:cobc
+	cmd:cobcrun
+	lib:libcob
+	"
+REQUIRES="
+	haiku
+	cmd:bison
+	cmd:flex
+	cmd:gcc
+	cmd:m4
+	cmd:perl
+	lib:libdb
+	lib:libgmp
+	lib:libidn2
+	lib:libintl
+	lib:libltdl
+	lib:libncurses
+	"
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libdb
+	devel:libgmp
+	devel:libidn2
+	devel:libintl
+	devel:libltdl
+	devel:libncurses
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:bison
+	cmd:flex
+	cmd:gcc
+	cmd:make
+	cmd:m4
+	cmd:perl
+	cmd:pkg_config
+	"
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure
+	make $jobArgs
+}
+INSTALL()
+{
+	make install
+	prepareInstalledDevelLib libcob
+	rm $developLibDir/*.la
+	fixPkgconfig
+
+	# devel package
+	packageEntries devel \
+		$developDir
+}
```

Comment:
Could you please check if it comes with some tests? If so, please add a TEST() section too. Do not forget about the newline before the TEST()

Reply:
No tests that I know of. Updated the rest as you commented.